dev-java/eclipse-ecj/files/ecj-4.27

```diff
@@ -0,0 +1,7 @@
+JAVAC="@GENTOO_PORTAGE_EPREFIX@/usr/bin/ecj-4.27"
+PACKAGE="=dev-java/eclipse-ecj-4.27*"
+SUPPORTED_TARGET="1.1 1.2 1.3 1.4 1.5 1.6 1.7 1.8 9 10 11 12 13 14 15 16 17 18 19"
+SUPPORTED_SOURCE="1.3 1.4 1.5 1.6 1.7 1.8 9 10 11 12 13 14 15 16 17 18 19"
```

Comment:
so you dropped this ebuild?

Reply:
> so you dropped this ebuild?

noo, just separated, saved for later in https://github.com/gentoo/gentoo/pull/35815

Reply:
> are those really supported only up to java 19?

for those ecl-{SLOT} files of all version it needs your help since i don't understand them at all. please adjust if something wrong.

Reply:
so i checked that and this seems to be fine:
https://github.com/eclipse-jdt/eclipse.jdt.core/blob/R4_27_maintenance/org.eclipse.jdt.core.compiler.batch/src/org/eclipse/jdt/internal/compiler/impl/CompilerOptions.java#L223-L243